config/kubesan/kubernetes/manager/cluster.yaml

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubesan-cluster-controller-manager
+  namespace: system
+  labels:
+    app.kubernetes.io/name: kubesan-cluster-controller-manager
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kubesan-cluster-controller-manager
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kubesan-cluster-controller-manager
+        app.kubernetes.io/part-of: ksan-storage
+        app.kubernetes.io/component: cluster-controller-manager
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+    spec:
+      serviceAccountName: cluster-controller-manager-kubesan
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      containers:
```

Comment:
same comment as above related to nodeSelector or affinity boundaries to prevent unsafe scheduling